app/controllers/api_controller/arbitration_rules.rb

```diff
@@ -0,0 +1,30 @@
+class ApiController
+  module ArbitrationRules
+    def create_resource_arbitration_rules(type, _id, data)
+      attributes = validate_arbitration_rules(data)
+      arbitration_rule = collection_class(type).create(attributes)
+      if arbitration_rule.invalid?
+        raise BadRequestError,
+              "Failed to create a new virtual template - #{arbitration_rule.errors.full_messages.join(', ')}"
+      end
+      arbitration_rule
+    end
+
+    def edit_resource_arbitration_rules(type, id, data)
+      attributes = validate_arbitration_rules(data)
+      edit_resource(type, id, attributes)
+    end
+
+    private
+
+    def validate_arbitration_rules(data)
+      if data.key?('id') || data.key?('href')
```

Comment:
@jntullo I think that's because in your test for passing the href you are sending a collection url as opposed to a resource - you may need to do `abribration_rules_url(999_999)`.